src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
```

Comment:
I don't think this historical information is necessary to have in the source code - and there have been long-standing feature requests already https://github.com/Mudlet/Mudlet/issues/685 and https://github.com/Mudlet/Mudlet/issues/741 that you're covering off with this :)